### ISSUE TITLE
Action fails when used with the suggested calling syntax

### ISSUE THREAD

**Issue by ant0nsc** [microsoft/security-devops-action#21]
On github marketplace, it suggests to include this action with the following snippet:
```
- name: security-devops-action
  uses: microsoft/security-devops-action@v1.5.0
```
However, doing that results in a workflow failure. I tried this [on my repo](https://github.com/microsoft/hi-ml/commit/d5cd1d3ad7a2a3139c736a9d50a32756ffe3b4d4), but the resulting [pipeline failed saying that Guardian wants --config or --tools](https://github.com/microsoft/hi-ml/runs/6553065874?check_suite_focus=true)

**Comment by markphip** [comment]
I am just checking this action out for first time and wanted to see what Issues are open .. keep that in mind. I will just note you are running your Action workflow on Ubuntu but the README says this action currently only works on Windows. So that could explain the error you are receiving. Try changing your job to run on `windows-latest` runner?

**Comment by johnemau** [comment]
I am [hitting the same failure](https://github.com/webhintio/hint/pull/5177/checks#step:2:35) and have the workflow configured to [run on `windows-latest`](https://github.com/webhintio/hint/pull/5177/commits/ab776dc527fe6b21b73267b719ab8b8b96f74498#diff-5b6df56c73f147cdd18362f90c286ac086e042bcb42b50bbf384321dcf21e1d6R14).

It looks like the issue is the action has no way of providing a path to a configuration file (either a generic config file or a tool specific config file). This _appears_ to be a required parameter to the command line the task is calling, however the task does not provide the argument.

I attempted to side-step the issue by providing a configuration file on disk with the hope it would be automatically discovered and picked up by the tooling, unfortunately that is not the case.

**Comment by ant0nsc** [comment]
Hi @markphip , unfortunately I get exactly the same error when running on `windows-latest`: https://github.com/microsoft/hi-ml/runs/6755283504?check_suite_focus=true

**Comment by markphip** [comment]
Yeah the previous comment said the same thing. I am also just a user so I do not have anything else to add.

**Comment by ivadim** [comment]
I can be wrong (just a user too), but you may need to add `checkout` step. I assume "autodetection" feature need files to determine the scans and create configs based on them.

**Comment by ant0nsc** [comment]
(facepalm) Thanks a ton @ivadim , that was precisely the problem
